@xen-orchestra/web-core/lib/components/ui/label/UiLabel.vue

```diff
@@ -1,18 +1,15 @@
-<!-- v1 -->
+<!-- v3 -->
 <template>
   <div :class="toVariants({ accent })" class="ui-label">
-    <VtsIcon :name="icon" size="medium" class="icon" />
     <label :for="htmlFor" :class="{ required }" class="typo-caption label">
```

Comment:
```suggestion
    <label :for="htmlFor" :class="{ required }" class="typo-caption label">
```
<img width="174" height="112" alt="image" src="https://github.com/user-attachments/assets/715eb1cd-8b0e-4fc9-9be2-7f06cb8624fc" />

According to the figma , we need to change the typography